TASK-51872: can't remove start date of a task

hthabti opened the pull request:
issue: start date can't be removed when clicking on 'none' label from the datepicker
fix: add else-if statement to handle user's input on the datepicker when the value of entered label = 'none'

houssemexo26 committed:
TASK-35704: Create FB maintenance and update projects versions/dependencies

houssemexo26 committed:
TASK-35704: Create FB maintenance and update projects versions/dependencies

hakermi committed:
MAINT-43333: Fix task drawer ui issue in changes action history (#711)

ISSUE: UI style adjustement
FIX: Adjust a css margin style property

hakermi committed:
MAINT-43333: Fix task drawer ui issue in changes action history (#711)

ISSUE: UI style adjustement
FIX: Adjust a css margin style property

montassarhamadi committed:
Task-43333: Regression - bad display(alignment-UI glitches) of task changes in change history drawer. (#721)

Problem: a bad alignment of the task changes in task's changes tab is displayed.
Fix: Task changes should be well displayed with aligned.

montassarhamadi committed:
Task-43333: Regression - bad display(alignment-UI glitches) of task changes in change history drawer. (#721)

Problem: a bad alignment of the task changes in task's changes tab is displayed.
Fix: Task changes should be well displayed with aligned.

montassarhamadi committed:
Task-43333: Regression - bad display(alignment-UI glitches) of task changes in change history. (#723)

Problem: a bad alignment of the task changes in task's changes tab is displayed.
Fix: Task changes should be well displayed with aligned.

montassarhamadi committed:
Task-43333: Regression - bad display(alignment-UI glitches) of task changes in change history. (#723)

Problem: a bad alignment of the task changes in task's changes tab is displayed.
Fix: Task changes should be well displayed with aligned.

hthabti committed:
Update TaskDrawer.vue

issue: start date can't be removed when clicking on 'none' label from the datepicker
fix: add else-if statement to handle user's input on the datepicker when the value of entered label = 'none'

ahamdi commented on task-management/src/main/webapp/vue-app/taskDrawer/components/TaskDrawer.vue:
```diff
@@ -482,6 +482,17 @@ export default {
+      } else if (value === 'none' && this.oldTask.startDate) {
+        this.task.startDate = null;
+        this.$taskDrawerApi.updateTask(this.task.id, this.task).then(() => {
+          this.oldTask.startDate = this.task.startDate;
```
Is this needed ?

rdenarie replied:
cc @hthabti Can you answer to Ali's question ? 
Thanks

ahamdi reviewed (changes requested):
Please update the PR title

Noussairhamrit committed:
Task-52342 : Tasks status not updated on the board when updating status just after assignment (#730)

* Task-52342 : Tasks status not updated on the board when updating status just after assignment

ISSUES : Tasks status ,priority start date and due date not updated on the tasks board just after assignment
FIX : to fix status problem i add event 'update-task-status'  on method 'updateTaskStatus' in TaskDrawer.vue then i  receive the event in TaskViewDashboard.vue ,
	to fix priority problem i just modified in   TaskViewDashboard.vue  when a call the event task-priority-updated ,
	to fix start date problem i add event 'task-start-date-updated'  on method 'updateTaskStartDate' in TaskDrawer.vue then i  receive the event in TaskViewDashboard.vue ,
	finally  to fix due date problem i just add event 'task-due-date-updated'  on method 'updateTaskStartDate' in TaskDrawer.vue

rdenarie committed:
Merge branch 'feature/maintenance' into TASK-51872-can't-remove-start-date-of-a-task

ahamdi committed:
Merge branch 'feature/maintenance' into TASK-51872-can't-remove-start-date-of-a-task

hthabti committed:
Update TaskDrawer.vue

rdenarie committed:
Merge branch 'feature/maintenance' into TASK-51872-can't-remove-start-date-of-a-task

rdenarie commented:
@ahamdi as you made a comment on the PR, can you check if it is ok for you now ?
Thansk
